### PR TITLE
PMM-4561 Content in logs.zip.

### PR DESCRIPTION
**Pull request by adivinho**

### CODE REVIEW

File: supervisord.conf [hunk 1]
```diff
@@ -4,7 +4,7 @@ username = dummy
 password = dummy
 
 [supervisord]
-logfile = /srv/logs/supervisord.log
+logfile = /var/log/supervisor/supervisord.log
```
Comment (AlekSi) [inline supervisord.conf]: We want to move all logs to `/srv/logs`, not do the opposite thing. See https://confluence.percona.com/x/TIeUAg

Comment (adivinho) [reply supervisord.conf]: Process control system logs aren't mentioned in the document. This fact was [pointed out](https://jira.percona.com/browse/PMM-4561?focusedCommentId=240937&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-240937) by our QA team.

Comment (AlekSi) [reply supervisord.conf]: It is mentioned now. I also added a comment to the ticket.

Please revert that change.

Comment (AlekSi) [reply supervisord.conf]: @hors we should also use `/srv/logs/supervisord.log` in Ansible playbooks.

Comment (AlekSi) [reply supervisord.conf]: Cross-reference to https://github.com/Percona-Lab/percona-images/pull/36#discussion_r319905466

Comment (adivinho) [reply supervisord.conf]: supervisord.log location is reverted.

File: supervisord.conf [hunk 2]
```diff
@@ -63,7 +63,7 @@ command =
         --config=/etc/grafana/grafana.ini
         cfg:default.paths.data=/srv/grafana
         cfg:default.paths.plugins=/var/lib/grafana/plugins
-        cfg:default.paths.logs=/srv/logs
+        cfg:default.paths.logs=/var/log/grafana
```
Comment (AlekSi) [inline supervisord.conf]: Sorry, I did not notice that. :( We should use `/srv/logs` there too. And that changed should be synced with Ansible too. /cc @hors

File: supervisord.conf [hunk 3]
```diff
@@ -76,7 +76,7 @@ startretries = 3
 startsecs = 1
 stopsignal = TERM
 stopwaitsecs = 300
-stdout_logfile = /srv/logs/grafana.log
+stdout_logfile = /srv/logs/grafana.startup.log
```
Comment (AlekSi) [inline supervisord.conf]: 👍

**Pull request merged**